School/classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor label
```

Comment:
- [ ] As requested by the previous code reviewer, please kindly correct the syntax error, re-write the attr_accessor properly by adding a `colon` before the label instance variable.